pkgs/tools/misc/figurine/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+let
+  version = "1.3.0";
+in
+buildGoModule {
+  pname = "figurine";
+  inherit version;
```

```suggestion
buildGoModule rec {
  pname = "figurine";
  version = "1.3.0";
```

Mainly style. Both versions of that section of code accomplish the same.

`rec` here is just way more common.

And needed for the `ldflags` suggestion down below :)